strings/base_events.h

```diff
@@ -510,6 +499,32 @@ WINRT_EXPORT namespace winrt
 
     private:
 
+        WINRT_IMPL_NOINLINE event_token add_agile(delegate_type delegate)
```

Comment:
>delegate_type [](http://example.com/codeflow?start=50&length=13)

Can this be `const&` to avoid a copy?  Or does that somehow defeat the foldability?

Reply:
Nevermind, you move out of it so you need a copy

Reply:
... and it's the unnamed temporary from the call to make_agile_delegate.